### PR TITLE
Reduce API pull requests to 1/sec for maximum_gitlab_api_requests_per_second

### PR DESCRIPTION
**Pull request by dnsmichi**
## Details:

- https://github.com/mvisonneau/gitlab-ci-pipelines-exporter/issues/253
- https://gitlab.com/gitlab-org/gitlab/-/issues/327919#note_555854856

## Tests

Screenshot 2021-04-20 at 13 20 12

### CODE REVIEW

**Comment by dnsmichi** [comment]
Modified the example settings after reviewing the docs on a global level: https://github.com/mvisonneau/gitlab-ci-pipelines-exporter/blob/main/docs/configuration_syntax.md

File: examples/quickstart/gitlab-ci-pipelines-exporter.yml [hunk 1]
```diff
@@ -2,6 +2,12 @@ gitlab:
   url: https://gitlab.com
   token: <your_token>
 
+# Global pull settings
+pull:
```
Comment (mvisonneau) [inline examples/quickstart/gitlab-ci-pipelines-exporter.yml:6]: and therefore same here 👍

**Review by mvisonneau — CHANGES_REQUESTED** [review]
thanks for the change @dnsmichi, I made a small comment about the examples definitions but otherwise I am happy to merge it.

File: examples/ha-setup/gitlab-ci-pipelines-exporter.yml [hunk 2]
```diff
@@ -5,6 +5,12 @@ gitlab:
 redis:
   url: redis://redis:6379
 
+# Global pull settings
```
[the remark this answers is not in the document]
Comment (dnsmichi) [reply examples/ha-setup/gitlab-ci-pipelines-exporter.yml]: Good point. It is redundant, and users who upgrade will not recognize. Thanks for the kind review, I have adjusted it. :)

File: examples/quickstart/gitlab-ci-pipelines-exporter.yml (hunk 1 above)
Comment (dnsmichi) [reply examples/quickstart/gitlab-ci-pipelines-exporter.yml]: Addressed in a new commit, thanks 👍

**Review by mvisonneau — APPROVED** [review]
thanks @dnsmichi!

**Pull request merged**